VideoWeb/VideoWeb/ClientApp/src/app/shared/footer/footer.component.scss

```diff
@@ -3,24 +3,37 @@
   flex-flow: column;
 }
 
-.govuk-footer__copyright-logo {
+.govuk-footer__copyright-logo,
+.govuk-footer__copyright-logo-scottish {
   display: inline-block;
   min-width: 125px;
   padding-top: 112px;
-  background-image: url('/assets/images/govuk-crest.png');
   background-repeat: no-repeat;
   background-position: 50% 0%;
   background-size: 125px 102px;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
 }
+
+.govuk-footer__copyright-logo {
+  background-image: url('/assets/images/govuk-crest.png');
+}
+
+.govuk-footer__copyright-logo-scottish {
+  background-image: url('/assets/images/scottish-crest-footer.png');
+}
+
 @media only screen and (-webkit-min-device-pixel-ratio: 2),
   only screen and (min-resolution: 192dpi),
   only screen and (min-resolution: 2dppx) {
   .govuk-footer__copyright-logo {
     background-image: url('/assets/images/govuk-crest-2x.png');
   }
+
+  .govuk-footer__copyright-logo-scottish {
+    background-image: url('/assets/images/scottish-crest-footer.png');
```

Comment:
Did you confirm with Neil that this image is ok? If you make the change I suggested above you could also take this out.

Reply:
yea I confirmed with Neil, at the moment we dont have a high res image so asked me to use the same image. I would still need this change right because otherwise in high res it would then show the uk crest?

Reply:
If you make the previous change I suggested so the background image is set by govuk-footer-background-uk or whatever you call it then it wouldn't for scottish ones.

Reply:
> A few small suggestions / observations. I agree that this should of been a directive it would massively reduce the overhead of maintaining this particularly from a test point of view. Handling the different locations could be done something like this.
> 
> ![image](https://user-images.githubusercontent.com/8798036/140537028-b9d01856-ea7d-4f00-90c1-525261b5b8df.png)
> 
> Then if anything needs to change it the maps can be modified. A default crest can be used where no location is provided.

Thats the problem with this, not all are actually using images, one of them is a svg and in another place we are using 

> A few small suggestions / observations. I agree that this should of been a directive it would massively reduce the overhead of maintaining this particularly from a test point of view. Handling the different locations could be done something like this.
> 
> ![image](https://user-images.githubusercontent.com/8798036/140537028-b9d01856-ea7d-4f00-90c1-525261b5b8df.png)
> 
> Then if anything needs to change it the maps can be modified. A default crest can be used where no location is provided.

few things here, not all are img elements, one of them is a svg, one of the element is using background image inside of a span, some of them are also using different class names. Some components use same crest for scottish venue but different ones for uk etc. things like that has sort of flagged out to me is that it would have ended in being a really complicated directive that would be very hard to maintain.